src/collective/ifttt/actions/easyform.py

```diff
@@ -3,16 +3,30 @@
 from collective.easyform.actions import Action
 from collective.easyform.actions import ActionFactory
 from collective.ifttt import _
+from interfaces.ifttt_easyform_adapter import IIFTTT
 from plone.supermodel.exportimport import BaseHandler
+from zope.interface import implementer
 
 
+@implementer(IIFTTT)
 class IFTTTTrigger(Action):
-    pass
+    """Action Executor for easyform IFTTT adapter"""
+
+    def __init__(self, **kw):
+        for i, f in IIFTTT.namesAndDescriptions():
+            setattr(self, i, kw.pop(i, f.default))
+        # TODO solve this error
```

Comment:
@datakurre I need your help here to fix this error.
Step to reproduce this error:
 - Add new easyform.
 - Add a new action and select IFTTT Trigger.

Reply:
@datakurre any clue to this error??

Reply:
@Shriyanshagro Possibly. Thank you for pinging me on this. I'll add better review for this pull now...